calico_cni_test.go

```diff
@@ -297,4 +308,61 @@ var _ = Describe("CalicoCni", func() {
 			})
 		})
 	})
+
+	Describe("ADD a container with ContainerID and DEL it with the same ContainerID", func() {
```

Comment:
How is this test different from the first test in this file? Do we really need both?